sql/src/value.rs

```diff
@@ -230,13 +230,18 @@ impl TryFrom<(&DataType, &str)> for Value {
                 let decoder = ValueDecoder {};
                 decoder.read_field(t, &mut reader)
             }
-            DataType::Nullable(inner) => {
-                if v == NULL_VALUE {
-                    Ok(Self::Null)
-                } else {
-                    Self::try_from((inner.as_ref(), v))
+            DataType::Nullable(inner) => match inner.as_ref() {
+                DataType::String => Ok(Self::String(v.to_string())),
```

Comment:
why string is special?

Reply:
Because string could be `"NULL"`, but other types can't.